typings/index.d.ts

```diff
@@ -0,0 +1,101 @@
+declare module '@paralleldrive/react-feature-toggles' {
+  import * as React from 'react';
+
+  interface Feature {
+    readonly name: string;
+    readonly isActive: boolean;
+    readonly dependencies?: ReadonlyArray<string>;
+  }
+
+  interface Query {
+    readonly ft: string;
+  }
+
+  interface Req {
+    readonly query: Query;
+  }
+
+  interface CurrentActiveFeatures {
+    readonly initialFeatures: ReadonlyArray<string>;
+    readonly req?: Req;
+    readonly search?: string;
+  }
+
+  function Feature(
+    inactiveComponent: any,
+    activeComponent: any,
+    name: string,
+    children?: (args: React.ReactNode) => React.ReactNode
+  ): JSX.Element;
+  export { Feature };
+
+  function FeatureToggles({
+    features
+  }?: {
+    readonly features?: ReadonlyArray<string>;
+    readonly children?: React.ReactNode;
+  }): JSX.Element;
+  export { FeatureToggles };
+
+  function getActiveFeatures(
+    features: ReadonlyArray<Feature>
+  ): ReadonlyArray<string>;
+  export { getActiveFeatures };
+
+  function getQueryFeatures(query?: Query): ReadonlyArray<string>;
+  export { getQueryFeatures };
+
+  function mergeFeatures(
+    currentFeatures: ReadonlyArray<string>, // tslint:disable-next-line:readonly-array
+    ...restOfFeatures: Array<ReadonlyArray<string>>
+  ): ReadonlyArray<string>;
+  export { mergeFeatures };
+
+  function removeFeatures(
+    currentFeatures: ReadonlyArray<string>,
+    features: ReadonlyArray<string>
+  ): ReadonlyArray<string>;
+  export { removeFeatures };
+
+  function isActive(
+    name: string,
+    currentFeatures: ReadonlyArray<string>
+  ): boolean;
+  function isActive(
+    name: string
+  ): (currentFeatures: ReadonlyArray<string>) => boolean;
+  export { isActive };
+
+  function getReqQueryFeatures(req?: Req): ReadonlyArray<string>;
+  export { getReqQueryFeatures };
+
+  function getBrowserQueryFeatures(search?: string): ReadonlyArray<string>;
+  export { getBrowserQueryFeatures };
+
+  function getCurrentActiveFeatures(
+    obj: CurrentActiveFeatures
+  ): ReadonlyArray<string>;
+  export { getCurrentActiveFeatures };
+
+  type ConfigureFeature1 = (
```

Comment:
Should I move these to the top or are they better down by the function?

Reply:
I think by the function is cool for those if they are only used by the one function.

Reply:
Ok that is what I was thinking. They are only used here.